src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -174,32 +174,26 @@ HttpStatusCode mongoUpdateContextSubscription
 
   }
 
-  /* Duration update */
-  long long expiration = -1;
-
-  if (requestP->expires != -1) //v2
-  {
-    expiration = requestP->expires;
-  }
-  else
+  /* Expiration */
```

Comment:
This is the fix for:

"Fix: csub cache at mongoBackend update context subscription logic wrongly setting expiration with current time, ignoring actual expires/duration in the request"